src/StatusQ/Components/StatusListPicker.qml

```diff
@@ -0,0 +1,400 @@
+   The drop-down list incorporates a searcher by the following model roles: `name` or / and `shortName`.
+
+   NOTE: Make sure to set the appropriate z-index when instantiating the component in order to place it on top of your view in order to display the drop-down data properly.
+
+   Example of how the component looks like:
+   \image status_list_picker.png
+   Example of how to use it:
+   \qml
+        StatusListPicker {
+            id: currencyPicker
+            inputList: Models.currencyPickerModel
+            searchText: qsTr("Search Language")
+            isRadioBtn: false
+            printSymbol: true
+        }
+   \endqml
+   For a list of components available see StatusQ.
+*/
+Item {
+    id: root
+
+    /*!
+       \qmlproperty string StatusListPicker::inputList
+       This property holds the data that will be populated in the list picker.
+
+       NOTE: It is an in / out property so the `selected` role will be dynamically changed according to the user actions.
+
+       Here an example of the model roles expected:
+       \qml
+            inputList: ListModel {
+            ListElement {
+                key: 0
+                name: "United States Dollar"
+                shortName: "USD"
+                symbol: "$"
+                imageSource: "../../assets/twemoji/26x26/1f4b4.png"
+                category: "fiat"
+                selected: true
+            }
+            ListElement {
+                key: 1
+                name: "British Pound"
+                shortName: "GBP"
+                symbol: "£"
+                imageSource: "../../assets/twemoji/26x26/1f4b5.png"
+                category: "fiat"
+                selected: false
+            }
+        }
+       \endqml
+
+
+    */
+    property ListModel inputList: ListModel { }
+
+    /*!
+       \qmlproperty string StatusListPicker::searchText
+       This property holds the placeholder text the searcher input displays by default.
+    */
+    property string searchText: qsTr("Search")
+
+    /*!
+       \qmlproperty string StatusListPicker::isRadioBtn
+       This property holds whether the list picker is a single or a multiple picker option by displaying a StatusRadioButton or StatusCheckBox instead.
+    */
+    property bool isRadioBtn: true
+
+    /*!
+       \qmlproperty string StatusListPicker::printSymbol
+       This property holds whether the selected items will display a composition of the role `symbol` in the StatusPickerButton text or just only the role `shortName`.
+    */
+    property bool printSymbol: false
+
+    /*!
+       \qmlproperty string StatusListPicker::maxPickerHeight
+       This property holds the maximum drop-down list height allowed. The drop-down list height will be set as the minimum value between the list content height and the maxPickerHeight property.
+    */
+    property int maxPickerHeight: 718
```

Comment:
Great catch. `maxPickerHeight` can be used to contain the popup in parent's/window boundaries so the user can access all the items by scrolling. I can't see how we control the popup position or width for the same use-case,  when it is spanned partially outside its parent view and obstruct checkboxes. A note in the doc header for a user to consider this on a case by case basis would do.

Reply:
Yeah, I decided only to expose the height because the width and position seemed to me too much but, yes, it could be considered as well.

What do you think if we add in the header something like this:
"Make sure to position the component inside its parent also when it is spanned in order to achieve a better user experience."

Reply:
Sounds great.